.github/actions/prepare/action.yml

```diff
@@ -5,12 +5,10 @@ name: Prepare
 runs:
   steps:
     - uses: pnpm/action-setup@a7487c7e89a18df4991f7f222e4898a00d66ddda # v4
-      with:
-        version: 9
     - uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4
       with:
         cache: pnpm
-        node-version: "22.12.0"
+        node-version: "20"
```

Comment:
🤔 This might not work. It might need to be synced to the package.json engine, or be customizable...

Reply:
https://github.com/JoshuaKGoldberg/create-typescript-app/pull/2100